src/main/java/com/puppycrawl/tools/checkstyle/api/AbstractCheck.java

```diff
@@ -240,13 +240,40 @@ public final void setTabWidth(int tabWidth) {
     /**
      * Helper method to log a LocalizedMessage.
      *
+     * <pre>
+     *     col = 1 + CommonUtils.lengthExpandedTabs(
+                getLines()[ast.getLineNo() - 1], ast.getColumnNo(), tabWidth)
+     * </pre>
```

Comment:
@romani Doesn't it seem weird we are duplicating and then documenting code in Javadoc?
javadoc should be user friendly description of method and nuances. If code in method changes, I'm sure no one will remember to update this specific javadoc.

To me it seems this documentation should be comments on the actual code inside the method.

Reply:
....... ok, agree, lets remove this code block.

Reply:
done